src/main/resources/templates/build.gradle.ftl

```diff
@@ -0,0 +1,98 @@
+/*
+ * This file was generated by the Gradle 'init' task.
```

Comment:
@tsegismont remind me soon to propose a Kotlin Gradle DSL template instead of the Groovy one. It works so much better in the IDE...